nbgitpuller/static/js/index.js

```diff
@@ -139,7 +142,8 @@ var gs = new GitSync(
     get_body_data('branch'),
     get_body_data('depth'),
     get_body_data('targetpath'),
-    get_body_data('path')
+    get_body_data('path'),
+    get_body_data('provider')
```

Comment:
Can we make this contentProvider as well instead of just provider? Staying consistent seems most important.

Reply:
I ran into problems here with the jinga template( I assume it is jinga) I couldn't find documentation on the naming rules for variables but contentprovider was not working. I can look around again -- yes no problem.